tests/conditional_compilation/test_config.yml

```diff
@@ -0,0 +1,15 @@
+# Using models from https://github.com/openvinotoolkit/testdata
+# $find models -wholename "*.xml"
+
+- model:
```

Comment:
Need to redesign test config to support several models in scope of one test case

Reply:
Let's make it separate commit